admin/controllers/check-form.js

```diff
@@ -297,22 +293,23 @@ const assignCheckFormToWindowPage = async (req, res, next) => {
   let checkWindow
 
   try {
-    checkWindow = await checkWindowDataService.fetchCheckWindow(checkWindowId)
+    checkWindow = await checkWindowDataService.sqlFetchCheckWindow(checkWindowId)
```

Comment:
It might be better to call it `sqlFindOneById` (assuming checkWindow.id is passed over) and should return an object not an array.  The CheckWindow part is duplicated as this is the Check Window data service.

`find()` - returns an array
`findOne()` - returns an object